src/pages/Overview/OverviewPage.tsx

```diff
@@ -210,7 +218,10 @@ export class OverviewPage extends React.Component<OverviewProps, State> {
           {
             type: type,
             namespaces: Sorts.sortFunc(allNamespaces, sortField, isAscending),
-            displayMode: displayMode
+            displayMode: displayMode,
+            showConfirmModal: false,
```

Comment:
I guess that this should be set to the previous value of this very same field. Otherwise, when the health is refreshed (via the drop-down (e.g. Every 10s) the modal is hidden.
Happen to me, if you are reading the modal more than 10s and the refresh is triggered, the modal is gone.

Reply:
> Happen to me, if you are reading the modal more than 10s and the refresh is triggered, the modal is gone.

Ah, true

Reply:
@xeviknal updated